Neptune/IMessage.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+
+namespace Neptune
+{
+
+    public interface IMessage<IS>
+        where IS:IService
+    {
+    }
+    public interface IResponse<IMessage>
```

Comment:
What are you planning on having this contain?

Reply:
Whatever the response body for the message is. If the message is HealthCheck, then the response body would possibly contain {"status": "green", "mem_usage": "4GB", ...}